motor_controller/motorbench-test-project/motorbench-test-project/rb_library/rb_measure.h

```diff
@@ -165,14 +167,24 @@ inline static int16_t RB_LPF(int16_t input, int16_t prevOutput, int16_t coeff)
 {
     int32_t stateVar;
     
-    stateVar = (prevOutput << 15); // load 16-bit int into 32-bit int
+    stateVar = (int32_t)(prevOutput); //(prevOutput << 15); // load 16-bit int into 32-bit int
     stateVar += __builtin_mulss(input, coeff);
     stateVar -= __builtin_mulss(prevOutput, coeff);
     
     return (int16_t)(stateVar >> 15); // remove extra 2^15 factor
     
 }
 
+void RB_FaultInit(RB_FAULT_DATA *state);
+
+bool phaseCurrentFault(MC_ABC_T *piabc);
```

Comment:
would be best practice for these to be static or prepended with RB_, just in case the same function is declared in some generated file

Reply:
prepended with RB_